### ISSUE TITLE
Building image from the source does not work

### ISSUE THREAD

**Issue by ttobis** [lastpass/lastpass-cli#669]
The last commit breaks the build: 
https://github.com/lastpass/lastpass-cli/commit/eb5efe4ea0e5143c4b764a905a4a6073c7dd2b3e#diff-dc0c348c5b9c44e9c0c7f57bac7e4e6296fd4b8d1fb8e7c7e2a92a1710472270R211 

Error message: 
```
lastpass-cli/endpoints.c:211:16: error: use of undeclared identifier 'url'
                             "url", url,
                                    ^
```

**Comment by bormosLP** [comment]
Sorry for the inconvenience, here is the fix #670.

**Comment by ttobis** [comment]
Thank you!
